src/hotspot/share/runtime/vmThread.cpp

```diff
@@ -130,12 +131,15 @@ void VMThread::create() {
 
   _terminate_lock = new Monitor(Mutex::nosafepoint, "VMThreadTerminate_lock");
 
-  if (UsePerfData) {
+  if (UsePerfData && os::is_thread_cpu_time_supported()) {
```

Comment:
Similarly, check for `os::is_thread_cpu_time_supported()` should be a inner nested if, to avoid breaking `_perf_accumulated_vm_operation_time`.